errbot/core_plugins/flows.plug

```diff
@@ -0,0 +1,10 @@
+[Core]
```

Comment:
haha indeed, never noted this.

core boolean notes that it is a core plugin for errbot (a plugin to manage errbot basically)

code section is a yaspy jargon to say this is the minimum mandatory section in the plugin descriptor